elements/pl-string-input/pl-string-input.py

```diff
@@ -27,16 +27,21 @@ def render(element_html, data):
     label = pl.get_string_attrib(element, 'label', None)
     suffix = pl.get_string_attrib(element, 'suffix', None)
     display = pl.get_string_attrib(element, 'display', 'inline')
-    remove_leading_trailing = pl.get_string_attrib(element, 'remove-leading-trailing', False)
-    remove_spaces = pl.get_string_attrib(element, 'remove-spaces', False)
+    remove_leading_trailing = pl.get_boolean_attrib(element, 'remove-leading-trailing', True)
+    remove_spaces = pl.get_boolean_attrib(element, 'remove-spaces', False)
```

Comment:
Should there also be help text that describes what `remove_spaces` will do, if this flag is `True`?

Reply:
done